### PR TITLE
Migrate icons to @iconify/svelte

### PR DESCRIPTION
**Pull request by janosh**
Plus update deps. In particular, bump `svelte-toc` and `svelte-multiselect`.

### CODE REVIEW

File: src/lib/ChapterList.svelte [hunk 1]
```diff
@@ -23,7 +20,7 @@
 </ol>
 {#if startingChapters.length > 2}
   <h1>
-    <Construction {style} />
+    <Icon icon="ic:round-construction" inline />
```
Comment (janosh) [inline src/lib/ChapterList.svelte:23]: @kleinicke Could you or someone from IT take over converting our SVG icons to `@iconify/svelte` as done here and in `src/lib/BasePage.svelte`?

File: src/lib/Header.svelte [hunk 2]
```diff
@@ -9,19 +10,19 @@
...
-  //   searchKey: import.meta.env.VITE_ALGOLIA_SEARCH_KEY,
-  //   loadingStr: `Suche läuft...`,
-  //   noResultMsg: (query: string) => `Keine Ergebnisse für '${query}'`,
-  //   resultCounter: (hits: unknown[]) =>
-  //     hits.length > 0 ? `<span>Ergebnisse: ${hits.length}<span>` : ``,
-  //   placeholder: `Suche`,
-  //   ariaLabel: `Suche`,
-  // }
+  const searchProps = {
+    indices: Object.fromEntries(
```
Comment (dominikgeissler) [inline src/lib/Header.svelte:14]: can we also extract these hardcoded strings to the `smallText.yml`?

**Review by dominikgeissler — COMMENTED** [review]
Looks good! Thanks!

**Comment by dominikgeissler** [comment]
Are the hooks supposed to fail? :D
Somehow the Lighthouse-Hook has no keys.
```
svelte-algolia: Invalid appId: undefined
svelte-algolia: Invalid searchKey: undefined
svelte-algolia: Invalid appId: undefined
svelte-algolia: Invalid searchKey: undefined
svelte-algolia: Invalid appId: undefined
svelte-algolia: Invalid searchKey: undefined
svelte-algolia: Invalid appId: undefined
svelte-algolia: Invalid searchKey: undefined
svelte-algolia: Invalid appId: undefined
svelte-algolia: Invalid searchKey: undefined
svelte-algolia: Invalid appId: undefined
svelte-algolia: Invalid searchKey: undefined
Error: missing Airtable API key, got undefined
```

File: src/lib/ChapterList.svelte (hunk 1 above)
Comment (kleinicke) [reply src/lib/ChapterList.svelte:23]: I'm not really sure what to do here. Should I figure out a corresponding way to call the same icon with the new framework?

**Pull request merged**